### ISSUE TITLE
http device: overlap with sandstorm bridge?

### ISSUE THREAD

**Issue by dckc** [Agoric/SwingSet#76]
who just suggested looking into this in FriAm? Was it @kpreid ?

**Comment by kpreid** [comment]
Yep. I was thinking that both projects could benefit from not separately designing and refining "turn external HTTP into structured capability-flavored messages and back".
